src/sparseml/yolov8/trainers.py

```diff
@@ -711,7 +712,13 @@ def export(self, **kwargs):
             else ["output0"],
         )
 
-        onnx.checker.check_model(os.path.join(save_dir, name))
+        complete_path = os.path.join(save_dir, name)
+        try:
+            skip_onnx_input_quantize(complete_path, complete_path)
+        except Exception:
+            pass
```

Comment:
@anmarques If the skip func above throws runtime error, wouldn't the check_model below fails? Is it safe to pass the exception?

Reply:
The Exception object contemplates RuntimeError